### PR TITLE
CHANGES for Chapel 1.27

### PR DESCRIPTION
**Pull request by bradcray**
This adds the list of CHANGES that will be in Chapel 1.27.  Thanks for
all the good work during this release cycle!

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -1,11 +1,300 @@
...
+---------------------------------
+* the compiler's LLVM back-end now supports version 14 as well as 11, 12, 13
+* made Chapel available as an Arch Linux AUR package  
+  (see https://aur.archlinux.org/packages/chapel)
+* updated `configure` to create prefix directories if they don't already exist
+* `make install` now installs `chpldoc` if it is built
+
+Semantic Changes / Changes to Chapel Language
+---------------------------------------------
+* made small integers prefer coercions to `real(32)` over `real(64)`
```
Comment (ronawho) [inline CHANGES.md:31]: coercions or coercing? (unsure myself)

File: CHANGES.md [hunk 2]
```diff
@@ -1,11 +1,300 @@
...
+
+Semantic Changes / Changes to Chapel Language
+---------------------------------------------
+* made small integers prefer coercions to `real(32)` over `real(64)`  
+  (see https://chapel-lang.org/docs/1.27/language/spec/conversions.html#implicit-numeric-and-bool-conversions)
+* made `public use` and `import` statements no longer introduce a hidden scope  
+  (see https://chapel-lang.org/docs/1.27/language/spec/modules.html#using-and-importing)
+* made `public use M` no longer bring in the name `M` for qualified access  
+  (see https://chapel-lang.org/docs/1.27/language/spec/modules.html#public-and-private-use-statements)
+* changed methods to no longer be subject to shadowing
```
Comment (ronawho) [inline CHANGES.md:37]: This reads a little odd to me

File: CHANGES.md [hunk 3]
```diff
@@ -1,11 +1,300 @@
...
+* `param` indexing of `string` and `bytes` values is now done at compile-time
+* changed one-tuples to print out as `(10,)` rather than `(10)` by default
+
+Namespace Changes
+-----------------
+* made internal module names not visible to Chapel programs by default
+
+Changes / Feature Improvements in Libraries
+-------------------------------------------
+* updated `.mod()`/`%` behavior for `bigint`s to match `mod()`/`%` for `int`s
```
Comment (ronawho) [inline CHANGES.md:70]: `.mod()` -> `mod()`? or is this the syntax we use to denote methods?

File: CHANGES.md [hunk 4]
```diff
@@ -1,11 +1,300 @@
...
+Changes / Feature Improvements in Libraries
+-------------------------------------------
+* updated `.mod()`/`%` behavior for `bigint`s to match `mod()`/`%` for `int`s  
+  (see https://chapel-lang.org/docs/1.27/modules/standard/BigInteger.html#BigInteger.bigint.mod)
+* improved `readLine` to return bytes read, throw errors, strip newlines, etc.  
+  (see https://chapel-lang.org/docs/1.27/modules/standard/IO.html#IO.channel.readLine)
+
+Name Changes in Libraries
+-------------------------
+* renamed `[channel.]readline()` to `[channel].readLine()`
```
Comment (ronawho) [inline CHANGES.md:77]: It's not obvious to me what this means. Made it a method instead of a standalone function?

Comment (ronawho) [reply CHANGES.md:77]: Oh, maybe it's just the camelCase switch, but the dot placement is different between the 2, which confused me.

File: CHANGES.md [hunk 5]
```diff
@@ -1,11 +1,300 @@
...
+Highlights (see subsequent sections for further details)
+--------------------------------------------------------
+* improved GPU support, including enabling the use of multiple locales
+* improved namespace behavior w.r.t. `import`, `use`, and shadowing
+* the Chapel compiler now supports LLVM 14, in addition to 11, 12, and 13
+* improved the performance and scalability of scan operations on `Block` arrays
+* added new 'OS.POSIX' and 'GPUDiagnostics' modules
+* restored support for Omni-Path networks
+* introduced a Code of Conduct for Chapel developers
+* made many changes, mostly minor, in support of Chapel 2.0 stabilization
```
Comment (ronawho) [inline CHANGES.md:18]: Is this the first mention of 2.0 in the CHANGES? Wondering if this needs more context or if we've talked about it enough in release notes?

File: CHANGES.md (hunk 1 above)
Comment (bradcray) [reply CHANGES.md:31]: I think either could work but I'm going to call this under my bar for today.  :)

File: CHANGES.md (hunk 3 above)
Comment (bradcray) [reply CHANGES.md:70]: Yeah, that was meant to imply a method.  I would've used `bigint.mod()` if I hadn't been trying to describe both mod and % in a single bullet within the one-line constraint.

Comment (bradcray) [reply CHANGES.md:70]: (also, note that the integer mod is not a method, which is the reason for the asymmetry).

File: CHANGES.md (hunk 4 above)
Comment (bradcray) [reply CHANGES.md:77]: Oops, yeah, good point.  It was meant to just be the camel-case difference (Applied to both the method and standalone versions of readline).  I'll fix the mistake with the placement of the `.`.

**Pull request merged**